src/components/InputDropdown/index.js

```diff
@@ -32,7 +32,7 @@ class InputDropdown extends React.Component {
   }
 
   componentWillUnmount() {
-    window.removeEventListener('mousedown')
+    window.removeEventListener('mousedown', () => false)
```

Comment:
I added a fix for this that might be closer to what was intended: https://github.com/helpsupply/helpsupply/pull/81/files/2b02fac50ca47940c2a0e4d6f86fe3b1fa5729d7#diff-ab45b32c1a60f1bea17b342d7592bc0c

What do you think?

Reply:
thank you!